sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/Generated/Models/Location.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.DeviceUpdate.Models
+{
+    /// <summary> The Location. </summary>
+    public partial class Location
```

Comment:
Can you give this model a more specific name? `Location` is too general, and we also don't like single word class name.

Reply:
This was automatically generated from the swagger. Are we allowed to edit these files?

Reply:
Please see my latest comment, thanks!